h/streamer/websocket.py

```diff
@@ -11,7 +11,7 @@
 from ws4py.websocket import WebSocket as _WebSocket
 
 from h import storage
-from h.streamer import filter
+from h.streamer.filter import FILTER_SCHEMA, SocketFilter
```

Comment:
Importing the module here is a bit gross to patch, and also conflicts with the python builtin `filter()`